spec/android_localize_helper_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper.rb'
+
+describe Fastlane::Helper::Android::LocalizeHelper do
+  describe 'get_library_version_from_gradle_config' do
+    it 'returns nil when gradle file is not present' do
+      allow(File).to receive(:exists?).and_return(false)
+      expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key')).to be_nil
+    end
+
+    it 'returns nil when the key is not present' do
+      test_file_content = <<~CONTENT
+        my-test-key-foo = 'foo'
+        my-test-key-bad = 'bad'
+        my-test-key = 'my_test_value'
+      CONTENT
+
+      allow(File).to receive(:exists?).and_return(true)
+      allow(File).to receive(:open).with('./build.gradle', 'r').and_yield(StringIO.new(test_file_content))
+      expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key-b')).to be_nil
+    end
+
+    it 'returns the key content when the key is present' do
+      test_file_content = <<~CONTENT
+        my-test-key-foo = 'foo'
+        my-test-key-bad = 'bad'
+        my-test-key = 'my_test_value'
```

Comment:
```suggestion
        my-test-key     =    'my_test_value'
```
Just to test more-than-one-space cases